### PR TITLE
fix: Progress Rehaul

### PR DESCRIPTION
**Pull request by harshilp24**
## Checklist
I have:
- [x] run the content through Grammarly
- [ ] linked to sample apps when relevant
- [x] added the meta description for each page in the PR
- [ ] minimized the callouts and added only when necessary
- [ ] added the `queryString` parameter to the Tabs (if used)
- [ ] masked PII in images. For example, login credentials, account details, and more
- [x] added images only when necessary
- [x] deleted the images that are no longer used for the updated pages in the PR
- [ ] followed the image file naming convention while renaming or adding new images. (Use lowercase letters, dashes between words, and be as descriptive as possible)
- [ ] used the `<figure/>` tag instead of a markdown representation for images
- [ ] added the `<figcaption/>` tag to add a caption to the image
- [ ] added the `alt` attribute in the `` tag
- [ ] verified and updated the cross-references or created redirect rules for the changed or removed content 
- [ ] reviewed and applied the style changes for UI formatting. For example, Bold the UI elements(Buttons on screen) used in the doc.

### CODE REVIEW

File: website/docs/reference/widgets/progress.md [hunk 1]
```diff
@@ -1,39 +1,121 @@
+---
+description: Learn how to use the Progress widget to visually track the progress of tasks or processes in your application.
+---
 # Progress
 
-The Progress widget indicates the progress of certain user-performed or system-triggered actions.
+This page provides information on using the Progress, which allows users to visualize the progress of certain user-performed or system-triggered actions.
```
Comment (danciaclara) [inline website/docs/reference/widgets/progress.md:6]:
```suggestion
This page provides information on using the Progress widget, which is used to visualize the progression of specific operations and user or system-triggered actions.

```

File: website/docs/reference/widgets/progress.md [hunk 2]
```diff
@@ -1,39 +1,121 @@
...
+These properties are customizable options present in the property pane of the widget, allowing users to modify the widget according to their preferences.
+
+
+### Basic
+
+#### Infinite loading `boolean`
+
+<dd>
+
+Enables the widget to enter an infinite loading state, which is beneficial when the progress values cannot be determined. This feature can be activated for queries or API calls that require time to retrieve data.
```
Comment (danciaclara) [inline website/docs/reference/widgets/progress.md:23]:
```suggestion
Enables the widget to enter an infinite loading state, which is beneficial when the progress values cannot be determined.For instance, this property can be enabled for queries that take a longer time to execute.
```

File: website/docs/reference/widgets/progress.md [hunk 3]
```diff
@@ -1,39 +1,121 @@
...
+
+</dd>
+
+#### Type `string`
+
+<dd>
+
+Allows you to select the desired format for the progress bar, providing options for either a linear or circular representation of progress.
```
Comment (danciaclara) [inline website/docs/reference/widgets/progress.md:31]: Options have to be specified separately as in the usual reference format

File: website/docs/reference/widgets/progress.md [hunk 4]
```diff
@@ -1,39 +1,121 @@
...
+
+
+</dd>
+
+#### Progress `number`
+
+<dd>
+
+Specify the percentage of progress to be displayed to the user, which can be dynamically updated using values retrieved from queries or JavaScript functions.
```
Comment (danciaclara) [inline website/docs/reference/widgets/progress.md:40]:
```suggestion
Specify the value of the progress indicator (in percentage). You can also use values retrieved from queries or JavaScript functions within the mustache operator `{{}}`.
```

Comment (danciaclara) [reply website/docs/reference/widgets/progress.md:40]: give an example for the JS part

File: website/docs/reference/widgets/progress.md [hunk 5]
```diff
@@ -1,39 +1,121 @@
...
+</dd>
+
+
+### General
+
+#### Number of steps `number`
+
+<dd>
+
+Specify the number of steps to break down the progress bar, dividing it into multiple parts with fixed percentages of progress. This configuration allows for better communication and supports only positive integers.
```
Comment (danciaclara) [inline website/docs/reference/widgets/progress.md:51]:
```suggestion
Specify the number of steps to break down the progress bar into multiple parts with fixed progress percentages. This property only supports positive integers.
```

File: website/docs/reference/widgets/progress.md [hunk 6]
```diff
@@ -1,39 +1,121 @@
...
-| **progress** | It shows the current progress of the progress bar in percentage. | `{{<progress-widget_name>.progress}}` |
-| **visible**  | Visibility of the progress bar widget.                           | `{{<progress-widget_name>.visible}}`  |
+</dd>
 
-### Styles
+#### Show result `boolean`
 
+<dd>
+
+Allows you to enable or disable the display of the current evaluated percentage as a number along with the progress in the widget.
```
Comment (danciaclara) [inline website/docs/reference/widgets/progress.md:73]:
```suggestion
Control the display of the evaluated percentage as a number along with the progress in the widget.
```

File: website/docs/reference/widgets/progress.md [hunk 7]
```diff
@@ -1,39 +1,121 @@
...
+
```
Comment (danciaclara) [inline website/docs/reference/widgets/progress.md:74]: You missed the **Counterclockwise** property for the Circular type progress bar

**Review by danciaclara — CHANGES_REQUESTED** [review]

File: website/docs/reference/widgets/progress.md (hunk 7 above)
Comment (harshilp24) [reply website/docs/reference/widgets/progress.md:88]: updated

**Review by danciaclara — APPROVED** [review]

**Pull request merged**